did/peer/peer2.go

```diff
@@ -0,0 +1,158 @@
+package peer
+
+import (
+	gocrypto "crypto"
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	"github.com/TBD54566975/ssi-sdk/crypto"
+	"github.com/TBD54566975/ssi-sdk/did"
+	"github.com/TBD54566975/ssi-sdk/did/resolver"
+	"github.com/TBD54566975/ssi-sdk/util"
+)
+
+// Method2 Method 2: multiple inception key without doc
+type Method2 struct {
+	KT     crypto.KeyType
+	Values []any
+}
+
+func (Method2) Method() did.Method {
+	return did.PeerMethod
+}
+
+// Resolve Splits the DID string into element.
+// Extract element purpose and decode each key or service.
+// Insert each key or service into the document according to the designated pu
+func (Method2) resolve(didDoc did.DID, _ resolver.ResolutionOption) (*resolver.ResolutionResult, error) {
+	d, ok := didDoc.(DIDPeer)
+	if !ok {
+		return nil, errors.Wrap(util.CastingError, "did:peer")
+	}
+
+	// The '=' at the end is an artifact of the encoding, and will mess up the decoding
+	// over the partials, so is removed.
+	// https://identity.foundation/peer-did-method-spec/index.html#generation-method
+	// ds := string(d)
+	// if ds[len(ds)-1] == '=' {
+	// 	d = DIDPeer(ds[:len(ds)-1])
+	// }
```

Comment:
Can this be deleted?

Reply:
I'm not familiar with the code so would prefer leaving it